staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -119,6 +119,14 @@ func (o *RecommendedOptions) ApplyTo(config *server.RecommendedConfig) error {
 	if err := o.Features.ApplyTo(&config.Config); err != nil {
 		return err
 	}
+	if err := o.EgressSelector.ApplyTo(&config.Config); err != nil {
+		return err
+	}
+	if feature.DefaultFeatureGate.Enabled(features.APIServerTracing) {
+		if err := o.Traces.ApplyTo(config.Config.EgressSelector, &config.Config); err != nil {
+			return err
+		}
+	}
```

Comment:
if you intend to use tracing for constructed remote authn/authz clients in the future, this should probably come before Authentication.ApplyTo / Authorization.ApplyTo as well so the TracingProvider is set up correctly, right?

Reply:
moved it up.  I'll add instrumentation for authn/z in a follow-up